radis/levels/partfunc.py

```diff
@@ -420,7 +420,7 @@ def at_noneq(
         #                               'rot_distribution={0}'.format(rot_distribution)
 
         # Check inputs, initialize
-        if overpopulation is None:
+        if overpopulation is None or overpopulation == {}:
```

Comment:
how is this required ? (`if overpopulation == {}: overpopulation = {}` ? )

Reply:
I think there is a minor inconsistency somewhere and overpopulation is already set to {} and then it raises an error. I can syntax it a better way